.github/workflows/pull-request-reviews.yml

```
@@ -12,9 +12,19 @@ jobs:
       SKIP: no-commit-to-branch
     steps:
       - uses: actions/checkout@de0fac2e4500dabe0009e67214ff5f5447ce83dd # v6.0.2
+      - name: Set up APT caching
+        uses: actions/cache@27d5ce7f107fe9357f9df03efb73ab90386fccae # v5.0.5
+        path: |
```

actionlint (1.7.12)

[error] unexpected key "path" for step to execute action. expected one of "continue-on-error", "env", "id", "if", "name", "timeout-minutes", "uses", "with"

(syntax-check)